paper/paper.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "archeryutils: a Python package for archery calculations"
+tags:
+  - Python
+  - Sport
+  - Archery
+authors:
+  - name: Jack Atkinson
+    orcid: 0000-0001-5001-4812
+    affiliation: 1,2
+    corresponding: true
+  - name: Thomas Hall
+    orcid: 0000-0003-1689-6676
+    affiliation: 1
+affiliations:
+  - name: Archery GB
+    index: 1
+  - name: Institute of Computing for Climate Science, University of Cambridge, UK
+    index: 2
+date: 21 April 2025
+bibliography: paper.bib
+
+---
+
+# Summary
+
+The sport of archery lends itself well to mathematical analysis and indeed there
+are several approaches to modelling performance across
+different distances and target types.
+
+_archeryutils_ is a Python package that provides base types and functions
+for developing code with archery applications, as well as implementations of popular
+mathematical models used in the sport.
+
+It is used in multiple applications including the development of websites
+and apps for archers, conducting performance analysis for elite archers and governing
+bodies, and in ongoing sports science research.
+
+
+# Statement of Need
+
+In the sport of archery athletes shoot targets at a variety of distances.
+Further, these targets can be of different sizes and have different scoring systems
+associated with them.
+The overall goal is always the same, however; to shoot arrows as accurately and
+precisely as possible.
+For the purposes of analysis, it is therefore useful to be able to extract some comparative
+measure of performance across all targets and distances.
+As a precision-based sport, archery lends itself well to mathematical
+modelling with examples in literature being @lane2013construction, @park2014modelling, and the
+2023 handicap system by Archery GB [@handicaps].
+Like many scientific models, these are described in papers where they are used by
+the authors, but no software implementation is publicly available.
+This presents a barrier to further research and development for those who understand
+how to use the models, but not how they are constructed or implemented.
```

Comment:
```suggestion
Although these models are well described in their respective papers, no software
implementations have been made publicly available, presenting a barrier to further
research and development.
```

Reply:
Add comment on reproducibility?

Reply:
Maybe change the end to `...presenting a barrier to reproducibility and limiting further research and development`? I think I only raised this suggestion as I had to re-read the original a couple of times to parse it, but that might just be a 'me' problem.

Reply:
No, it _was_ verbose.
I think I decided the reproducibility side was covered elsewhere and tidied up in https://github.com/jatkinson1000/archeryutils/pull/126/commits/8ffb87761fae084a5a94f80b27c705c4536a7e6f